docs/dev-deployment-README.md

```diff
@@ -0,0 +1,35 @@
+# AWS setup
+
+## Credentials
+
+Make sure you have sufficient AWS permissions for the actions required. If you have trouble, contact Jay.
+
+## AWS RDS
+
+1. Create AWS RDS database (if it is not already created). If it is created, make sure it is running.
+1. When creating it, choose the following options:
+   - `PostgreSQL - 12.5-R 1` for the Engine Type
+   - Template - `Free Tier` - for the dev stuff
+   - Master username - `postgres`
+   - Auto generate a password - it will show the generated password once the database is created. MAKE SURE TO SAVE IT!
+   - Leave the rest of the options as default
+1. The RDS should now be in creating mode. The credentials (Password) will be displayed. MAKE SURE TO SAVE IT in a safe place!
+1. Also note down the endpoint once available.
+1. Update the security group to allow inbound traffic from the SG that the EC2 instance belongs to (next section).
+
+## AWS EC2
+
+1. Create an AWS EC2 instance (ask Jay for more details). Make sure the public IP is selected, or else it won't be accessible from outside.
+1. Change the security group rules to allow Inbound traffic (Custom TCP) to Port 8888 from `Anywhere`
+1. Copy the latest version of the code to the EC2 instance using `scp`
```

Comment:
Worthwhile to mention the explicit need for copying over the images / other local database files or is a general "copy the code" cover that?

Reply:
Yup, copying the code covers that because the images, local assets are within the same directory!